backend/tests/integrated/infrastructures/load/adapters/test_repositories.py

```diff
@@ -0,0 +1,139 @@
+import pytest
+from django.db import DatabaseError
+from pytest_mock import MockFixture
+
+from infrastructures.apps.common import exceptions as common_exceptions
+from infrastructures.apps.load import models
+from modules.common import pagination as pagination_dtos
+from modules.load.domain import ports, value_objects
+from modules.load.services import queries
+from modules.load.services.queries import ports as query_ports
+from modules.load.services.queries import queries as query_dtos
+from modules.load.services.queries.ports import repositories as query_repositories
+from tests.model_factories import DataFactory
+
+from .....fakers import fake_transformed_data
+
+
+def test_django_data_domain_repository_create_method_creates_records(
+    test_django_data_domain_repository: ports.AbstractDataDomainRepository,
+):
+    # Given
+    records_number = 5
+    data = [fake_transformed_data() for _ in range(records_number)]
+
+    # When
+    test_django_data_domain_repository.create(data)
+
+    # Then
+    assert models.Data.objects.filter().count() == records_number
+    assert all(
+        models.Data.objects.filter(
+            data__contains={"full_name": data.full_name}
+        ).exists()
+        for data in data
+    )
+
+
+def test_django_data_domain_repository_create_method_raises_custom_exception_on_database_exception(
+    mocker: MockFixture,
+    test_django_data_domain_repository: ports.AbstractDataDomainRepository,
+):
+    # Given
+    side_effect = DatabaseError
+    mocker.patch(
+        "infrastructures.apps.load.models.Data.objects.bulk_create",
+        side_effect=side_effect,
+    )
+
+    # When and then
+    with pytest.raises(common_exceptions.DatabaseError):
+        test_django_data_domain_repository.create([fake_transformed_data()])
+
+    assert not models.Data.objects.exists()
+
+
+def test_django_data_query_repository_list_method_queries_all_records(
+    test_django_data_query_repository: query_repositories.AbstractDataQueryRepository,
+):
+    # Given
+    data_number = 3
+    DataFactory.create_batch(size=data_number)
+
+    # When
+    results, count = test_django_data_query_repository.list(
+        filters=query_ports.DataFilters(),
+        ordering=query_ports.DataOrdering(),
+        pagination=pagination_dtos.Pagination(
+            offset=pagination_dtos.PAGINATION_DEFAULT_OFFSET,
+            records_per_page=pagination_dtos.PAGINATION_DEFAULT_LIMIT,
+        ),
+    )
+
+    # Then
+    assert count == data_number
+    assert isinstance(results, list)
+    assert all(isinstance(result, query_dtos.OutputData) for result in results)
+
+
+def test_django_data_query_repository_list_method_raises_custom_exception_on_database_exception(
+    mocker: MockFixture,
+    test_django_data_query_repository: query_repositories.AbstractDataQueryRepository,
+):
+    # Given
+    side_effect = DatabaseError
+    mocker.patch(
```

Comment:
not like this.
Please, use patch.object and explicitly pass imported function with .name.
With that, once someone changes function name, it;s clear why this test fails.